src/components/field/index.js

```diff
@@ -651,13 +692,14 @@ export default class TextField extends PureComponent {
         onContentSizeChange={this.onContentSizeChange}
         onFocus={this.onFocus}
         onBlur={this.onBlur}
-        value={value}
+        value={this.props.defaultValue}
```

Comment:
This seems wrong. Doesn't this mean that the value of the `Input` won't ever update?

Reply:
Hm, yeah, I have the same question here.

Reply:
It seems to break when not using defaultValue, not sure what should be used here.